src/components/Mailbox.vue

```diff
@@ -162,8 +171,8 @@ export default {
 					this.$router.replace({
 						name: 'message',
 						params: {
-							accountId: this.account.id,
-							folderId: this.folder.id,
+							accountId: this.$route.params.accountId,
+							folderId: this.$route.params.folderId,
```

Comment:
I'm just curious: why were these changed from accessing the values directly on `this` to using the route params, while the other uses were changed the other way around? I'm not familiar with the code yet, so just trying to grok things 😅

Reply:
Hm. We definitely need it for the correct routing. But for the `fechEnvelope` call I'm not so sure anymore. I have to have another look :grin: